### PR TITLE
Set the bounds on the Autocomplete object

### PR DESCRIPTION
**Pull request by earayo**
It allows searching places near to the user. The location is set from browser.

### CODE REVIEW

**Comment by esbanarango** [comment]
@earayo Thank you for taking the time to add this feature, but, @dmuneras do we really want this behaviour by default? This should be optional, so we should add an option to active this. It's kind of annoying always showing this: 
promp 
by default.

**Comment by dmuneras** [comment]
I agree, I think we have to fix some small details until start adding new features.  @earayo  anyway, you can use this pointing the addon source to your forked repo, if you need it right now.

Example (thanks @esbanarango )

`"semantic-ui-ember": "git+https://github.com/esbanarango/Semantic-UI-Ember.git"`

**Comment by earayo** [comment]
@dmuneras and  @esbanarango Thanks for their feedback. Can I propose attaching a flag to indicate whether the developers want to include this behavior in the autocomplete?

**Comment by esbanarango** [comment]
@earayo Sure, go ahead and add that flag. Make sure to also add this to the readme.

Thank you.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ using a callback function in your controller.
 
 ![Autocomplete](http://i.imgur.com/wzGgfiY.gif)
 
+## Thanks!
```
Comment (esbanarango) [inline README.md]: @earayo Could you remove this section of the readme pls? We will add an `Authors` sections later. Thank you.

File: addon/components/place-autocomplete-field.js [hunk 2]
```diff
@@ -10,11 +10,34 @@ export default Component.extend({
...
+        let google = this.get('google') || window.google;
+        var geolocation = {
+          lat: position.coords.latitude,
+          lng: position.coords.longitude
+        };
+        var circle = new google.maps.Circle({
+          center: geolocation,
+          radius: position.coords.accuracy
+        });
+        this.get("autocomplete").setBounds(circle.getBounds());
```
Comment (esbanarango) [inline addon/components/place-autocomplete-field.js]: @earayo could you use single quotes here?

File: addon/components/place-autocomplete-field.js [hunk 3]
```diff
@@ -10,11 +10,34 @@ export default Component.extend({
...
+      });
+    }
+  },
 
   didInsertElement() {
     this._super(...arguments);
     this.getAutocomplete();
     this.get('autocomplete').addListener('place_changed', this.placeChanged.bind(this));
+    if (this.get("withGeoLocate")) {
```
Comment (esbanarango) [inline addon/components/place-autocomplete-field.js]: @earayo could you use single quotes here?

**Comment by dmuneras** [comment]
@esbanarango  don't merge this for now, I am updating the addon, I am going to finish the update by tomorrow.

**Comment by dmuneras** [comment]
@earayo we make some updates in the addon, could you rebase this PR against master? then we can review again and be able to merge this nice feature.

**Comment by earayo** [comment]
@esbanarango @dmuneras I just did rebase. They could review again. Thanks!

**Comment by esbanarango** [comment]
Thank you @earayo ! 👍

**Pull request merged**